scripts/nasa/cmip6_sea_level/process.py

```diff
@@ -0,0 +1,312 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import csv
+import glob
+import os
+import sys
+
+import numpy as np
+import xarray
+from absl import app
+from absl import flags
+
+# Allows the following module imports to work when running as a script
+_SCRIPT_PATH = os.path.dirname(os.path.abspath(__file__))
+sys.path.append(os.path.join(_SCRIPT_PATH, '../../../util/'))  # for recon util
+
+import latlng_recon_service
+import latlng_recon_geojson
+
+FLAGS = flags.FLAGS
+
+flags.DEFINE_string('in_pattern', '', 'Input NetCDF4 file(s).')
+flags.DEFINE_string('out_dir', '/tmp', 'Output directory path.')
+flags.DEFINE_string('generate_what', 'stat',
+                    'What to generate: place, sv, stat')
+
+###
+### Potentially general NetCDF4 processing config/code
+###
+
+## Fields for the config
+_MMETHOD = 'mmethod'  # Value is the measurement method
+_NCVAR = 'ncvar'  # Value is variable name in NetCDF4 file
+_PV = 'pv'  # Value is a list of PVs in MCF format
+_SVID = 'svid'  # Value is a pair of (SV ID part, rank)
```

Comment:
also describe the ranking (how it's defined and used?)

Reply:
Done.